frontend/src/components/ImplementationReportView/ImplementationReportView.tsx

```diff
@@ -38,12 +39,20 @@ const ReportComponent: React.FC<{ implementation: Implementation }> = ({
   return (
     <Container className="p-4">
       <Card className="mx-auto mb-3 col-md-9">
-        <Card.Header>
-          <span className="px-1 text-muted">
-            {mapLanguage(implementation.language)}
+        <Card.Header className="d-flex justify-content-between">
+          <span className="d-flex align-items-center">
+            <span className="px-1 text-muted">
+              {mapLanguage(implementation.language)}
+            </span>
+            <span>{implementation.name}</span>
+          </span>
+          <span>
+            {Object.entries(implementation.results).length !== 0 ? (
```

Comment:
Does this ever really happen that it's empty?

Reply:
> Does this ever really happen that it's empty?

I was not sure about it. I took precautions so that UI never breaks.

Reply:
It sounds like things are already broken if we ever have 0 results.

Reply:
I'll take out the unnecessary safety precaution.